### ISSUE TITLE
--outdir produces wrong output path (since v1.4.0)

### ISSUE THREAD

**Issue by aonyxrocks** [lustre-labs/dev-tools#58]
I am building my app like so:

`gleam run -m lustre/dev build app --minify=true --outdir="publish/js"`

Expectation:

```
[...]
✅ Bundle produced at `./publish/js/app.min.mjs`
```

Actual result:

```
[...]
✅ Bundle produced at `./publish/js/app.min../publish/js`
```

This started with version 1.4.0 and is still the case with 1.4.1. The last known version to work is 1.3.4.

**Comment by hayleigh-dot-dev** [comment]
Huh super odd, thanks i'll take a look.

**Comment by hayleigh-dot-dev** [comment]
Fixed in v1.4.2!
